### ISSUE TITLE
is there any way to avoid telescope picker for follow link

### ISSUE THREAD

**Issue by Aneeqasif** [renerocksai/telekasten.nvim#272]
the telescope picker pops up on  places where i dont need it to. like follwoing a link ceating daily/weekly notes etc any way to disable this

**Comment by lambtho12** [comment]
You can set the option `journal_auto_open = true' in the configuration. I think this is maybe not properly documented though.

This is actually a more or less a duplicate of https://github.com/renerocksai/telekasten.nvim/issues/93.

**Comment by Aneeqasif** [comment]
> You can set the option `journal_auto_open = true' in the configuration. I think this is maybe not properly documented though.
> 
> This is actually a more or less a duplicate of #93.

will this also stop telescope popup when following a link?

**Comment by pmatulis** [comment]
I also find it surprising that when you say to open a page that you are presented with a prompt for opening the page.

**Comment by Aneeqasif** [comment]
> It depends on your workflow. Often you just need to quickly skim a given note to retrieve a single piece of information. With the current implementation, you can just have a look in the picker without needing to open the file in a new buffer. This is why it was implemented this way.

sir many users are on small screens and its hard to skim through from small telescope window. so have an option just to turn it on off wont hurt anyone imo

**Comment by lambtho12** [comment]
@Aneeqasif, I agree this feature would definitely be useful. I was just explaining the reasoning behind the initial implementation :wink: 

It is definitely on my list regarding the refactoring of the pickers.

**Comment by Aneeqasif** [comment]
> It is definitely on my list regarding the refactoring of the pickers.

thank you sir for understanding. :heart:
